src/CFamily.UnitTests/CMake/CompilationDatabaseLocatorTests.cs

```diff
@@ -67,7 +71,7 @@ public void Locate_NoCMakeSettings_ReturnsDefaultLocationIfFileExists(bool fileE
         {
             var activeConfiguration = "any";
             var configProvider = CreateConfigProvider(activeConfiguration);
-            var cmakeSettingsProvider = CreateCmakeSettingsProvider(null);
+            var cmakeSettingsProvider = CreateCMakeSettingsProvider(RootDirectory, null);
```

Comment:
Miscellaneous: the `CreateCMakeSettingsProvider` was hard-coded to use a specific root directory, which would cause problems if it didn't match the root directory value that was passed to `CreateTestSubject`.